### DISCUSSION TITLE
Cannot update from 287 on Debian 11 Bullseye

### DISCUSSION THREAD

**Discussion by mantis64** [cockpit-project/cockpit#20429] [General]
### Explain what happens

1.  I want to update to the latest version
2.  I have done the following command:

. /etc/os-release
echo "deb http://deb.debian.org/debian bullseye-backports main" > \
    /etc/apt/sources.list.d/backports.list
apt update

3.  And then apt install -t bullseye-backports cockpit

I have made sure backports are enabled in my sources.list like below:

deb http://deb.debian.org/debian bullseye-backports main

However I always get that the latest version is 287 and I cannot upgrade.  Please can you help me upgrade to the latest version.  

### Version of Cockpit

287

### Where is the problem in Cockpit?

Software updates

### Server operating system

Debian

### Server operating system version

Debian 11 Bullseye

### What browsers are you using?

Chrome

**Comment by martinpitt** [comment]
This is intended. The current Debian release is 12 "bookworm", which receives updates/backports. 11 is "oldstable", and we don't support that any more with current Cockpit releases. Note that Debian 11 will be [end-of-life in a few months](https://wiki.debian.org/DebianReleases#Production_Releases).

**Comment by mantis64** [comment]
Thank you for replying and the clarification.  I have upgraded from 11 to 12 and was able to update my cockpit as expected.  

Thanks again!
